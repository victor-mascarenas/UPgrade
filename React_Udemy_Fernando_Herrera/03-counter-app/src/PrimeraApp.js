import React from "react";

//const PrimeraApp = (props) => {
const PrimeraApp = ({saludo}) => {
    return (
        <div>
            {/* <h1>{props.saludo}</h1> */}
            <h1>{saludo}</h1>
            <p>Mi primera aplicacion</p>
        </div>
    );
}

export default PrimeraApp;