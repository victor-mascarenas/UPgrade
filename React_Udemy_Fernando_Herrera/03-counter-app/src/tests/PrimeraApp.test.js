import React, { render } from "@testing-library/react";
import {shallow} from 'enzyme';
import PrimeraApp from "../PrimeraApp";

describe('Pruebas ne <PrimeraApp/>', () => {
    /* test('Debe de mostrar el mensaje "Hola soy Victor"', () => {
        const saludo = 'Hola osy Victor';
        //const wrapper = render(<PrimeraApp saludo={saludo}/>);
        //wrapper.getByText();
        const {getByText} = render(<PrimeraApp saludo={saludo}/>);
        expect(getByText(saludo)).toBeInTheDocument();
    }); */
    test('Debe de mostrar <PrimeraApp/> correctamente', () => {
        const saludo = 'Hola soy Victor';
        const wrapper = shallow(<PrimeraApp saludo={saludo}/>);
        expect(wrapper).toMatchSnapshot();
    });
});
