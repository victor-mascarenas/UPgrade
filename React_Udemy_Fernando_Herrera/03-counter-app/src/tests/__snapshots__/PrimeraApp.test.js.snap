// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas ne <PrimeraApp/> Debe de mostrar <PrimeraApp/> correctamente 1`] = `
<div>
  <h1>
    Hola soy Victor
  </h1>
  <p>
    Soy un subtitulo
  </p>
</div>
`;
